pkg/registry/servicecatalog/binding/strategy.go

```diff
@@ -90,6 +90,11 @@ func (bindingRESTStrategy) PrepareForCreate(ctx genericapirequest.Context, obj r
 	}
 	// Is there anything to pull out of the context `ctx`?
 
+	// If not specified, make the SecretName default to the binding name
```

Comment:
This goes in `pkg/apis/servicecatalog/v1alpha1/defaults.go` instead of the strategy.